### ISSUE TITLE
Problem with saving some track configuration

### ISSUE THREAD

**Issue by Cele91** [jo3stevens/ACServerManager#31]
I'm having several problems in saving some tracks configurations. If I select ks_vallelunga, and the "club" configuration, the panel writes "club" in cfg, but if I refresh the page in the panel appear "classic" , but in the cfg there are "club"

**Comment by Bechrissed** [comment]
Yes, same problem here. So the title of the issue is incorrect. It does save the values correctly in the cfg files but doesn't show those changes in the web interface. So if you save again you will be saving to a state you didn't choose. Quite annoying, it means you have to configure everything correct at once.

**Comment by Pringlez** [comment]
There seems to be an issue with the two way data binding, feel free to debug the issue further. I'll look into when I can

**Comment by Pringlez** [comment]
This is fixed in the latest pull request :+1:
